### ISSUE TITLE
Localize this extension to your own native language and region

### ISSUE THREAD

**Issue by Ceilican** [OiWorld/CarbonFootprintGoogleMaps#12]

**Comment by rohankatyal29** [comment]
@Ceilican: Are you expecting something like this: 

<img width="1273" alt="screen shot 2016-03-03 at 1 48 30 pm" src="https://cloud.githubusercontent.com/assets/2452426/13488358/ba5c5e80-e146-11e5-8f45-e67d299efd29.png">

A select menu to chose the language

**Comment by Ceilican** [comment]
Not exactly. Ideally, the app should detect the user's language automatically. This should be feasible with use Google's localization features. In particular, I was thinking of this: https://developer.chrome.com/webstore/i18n

Another app of mine that has been partially localized is [Mind The Word](https://github.com/OiWorld/MindTheWord) (see also https://github.com/OiWorld/MindTheWord/tree/master/Source/_locales and https://github.com/OiWorld/MindTheWord/blob/master/Description.md). However, even there I haven't been able to localize the texts that appear in the options page. I was only able to localize the texts that appear in the Chrome Web Store. Google's instructions don't say how to use [localized messages](https://github.com/OiWorld/MindTheWord/tree/master/Source/_locales) inside the app itself, and I haven't had time to research how to do that yet. If you could find that out, that would be great.

**Comment by rohankatyal29** [comment]
I have figured out a solution for this. But I am not being able to test it. I changed my chrome's locale using **chrome://setting/languages** and the used **var language = window.navigator.userLanguage || window.navigator.language;** to check my current locale, but it still shows en-US. I am not being able to change the accept language. I even tried with a chrome extension which changes the locale of browser with no luck. Do you know of any reliable way to change and test the locale of the browser?

**Comment by Ceilican** [comment]
No, unfortunately I don't know yet how to detect the user's language.

What you described seems to be a bug in Chrome. Maybe it would be worth it to send a bug report to Chrome's developers.

**Comment by rohankatyal29** [comment]
I will send a bug report to Chrome's developers and wait for their response

**Comment by share-with-me** [comment]
@rohankatyal29  I used navigator.userLanguage , navigator.browserLanguage , navigator.systemLanguage and managed to get the same output en-US . However, when I tried to change my browser default language to some other language, it still outputs en-US. Is it the same thing that you have tried? Also any updates from chrome bug issue?

**Comment by rohankatyal29** [comment]
@share-with-me: I have tried doing that plus a couple of things more with no success. Don't have any updates about the bug yet

**Comment by PrateekGupta1509** [comment]
I have implemented this in my pull request #33 , added support for Hindi language.  To add support for other languages would need their respective translations.

**Comment by Ceilican** [comment]
Thank you, @PrateekGupta1509 ! I will keep this issue open, in case other people would like to contribute other languages.

**Comment by AnandKumar22** [comment]
@ceilican I have added the Portugal Translation,  please have a look at it.
